cypress/integration/inventory/quick-marc.spec.js

```diff
@@ -90,7 +91,6 @@ describe('ui-inventory: MARC', () => {
 
     describe('editing the 006/00 field', () => {
```

Comment:
I can't open edit page because of disabled edit button to rewrite tests by using interactors